data/mods/Aftershock/items/inactiverobot.json

```diff
@@ -430,15 +430,20 @@
     "bashing": 6,
     "symbol": ";",
     "color": "dark_gray",
-    "flags": [ "LEAK_DAM", "RADIOACTIVE", "TRADER_AVOID" ]
+    "flags": [ "TRADER_AVOID" ]
```

Comment:
Does trader avoid mean the trader won't buy or sell it or just that they won't buy it?

Reply:
Prevents npcs with random inventories from starting witht them.